source/codegen/metadata/nixnet/functions.py

```diff
@@ -222,6 +224,14 @@
             {
                 'direction': 'out',
                 'name': 'numberOfBytesReturned',
+                'type': 'u32',
+                'include_in_proto': False
+            },
+            {
```

Comment:
note: Additional `frame_type` parameter required only when API returns a frame. This helps server identify how to interpret the `void*` buffer returned by C API.